WooCommerce/Classes/Extensions/UILabel+SalesChannel.swift

```diff
@@ -23,5 +23,7 @@ private extension UILabel {
     enum Layout {
         static let borderWidth = CGFloat(0.0)
         static let cornerRadius = CGFloat(4.0)
+        static let salesChannelLabelTextColor = UIColor(named: "posPrimary")
+        static let salesChannelLabelBackgroundColor = UIColor(named: "posSecondary")
```

Comment:
Another option here was to import SwiftUI and use `Color` directly. Thoughts? 🤔

Reply:
Good question. If we move away from POS to a separate model, such access could create problems. Also, this is not POS.  I would say we should use the appropriate Woo colors here, that is more common in the Woo store management part of the app.

-----

If you disagree, we at least should make it strongly typed:

```suggestion
        static let salesChannelLabelTextColor = UIColor(.posPrimary)
        static let salesChannelLabelBackgroundColor = UIColor(.posSecondary)
```

Reply:
> If you disagree

I agree, they feel a bit off with the rest of the colors. I'll check with Wagner to confirm them 👍 

> we at least should make it strongly typed

Thanks! Yeah, I think Xcode trolled me here as it wasn't compiling by using the strongly-typed reference, but now it does... that's why initially I thought about importing SwiftUI and such.